go.mod

```diff
@@ -4,16 +4,19 @@ go 1.13
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/aquasecurity/trivy v0.1.6
```

Comment:
Just curious what was this for?

Reply:
This change.
https://github.com/aquasecurity/trivy-db/commit/c9031b042c04d039492896a9b8d0bca7a97e5d30#diff-3661dc32b6bde1b0939ad409364e48fcL11

Reply:
Thanks! I was just curious.